app/services/user_service.py

```diff
@@ -0,0 +1,37 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class User:
+    email: str
+    slack_username: str
+    github_username: str
+
+
+class UserService:
+    def __init__(self, stubbed_users=None):
+        if stubbed_users is None:
+            stubbed_users = []
+        self.__stubbed_users = stubbed_users or [
```

Comment:
using stubbed_users at the moment - just to prove the model works with the service

It will be replaced with a Database Client Layer to fetch actual data later